pkgs/development/python-modules/txredisapi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub, nixosTests, six, twisted }:
+
+buildPythonPackage rec {
+  pname = "txredisapi";
+  version = "1.4.7";
+
+  src = fetchFromGitHub {
+    owner = "IlyaSkriblovsky";
+    repo = "txredisapi";
+    rev = "1.4.7";
+    sha256 = "1f7j3c5l7jcfphvsk7nqmgyb4jaydbzq081m555kw0f9xxak0pgq";
+  };
+
+  propagatedBuildInputs = [ six twisted ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled?

Reply:
Because there are no tests. The only tests are the unit tests ran in the nixos test

Reply:
I didn't want you to tell me that but rather add a comment with that information.

Reply:
Oh, okay. In the future maybe you could say something like "You should add a comment explaining why the tests are disabled", thank you though